data/json/professions.json

```diff
@@ -4232,5 +4232,65 @@
     "points": -3,
     "items": { "both": [ "shorts", "heroin", "syringe" ], "male": [ "briefs" ], "female": [ "bra", "panties" ] },
     "addictions": [ { "intensity": 40, "type": "opiate" } ]
+  },
+  {
+    "type": "profession",
+    "ident": "yoga_instructor",
+    "name": "Yoga Instructor",
+    "description": "You were in the middle of downward dog. Now you need to get down and out.",
+    "points": 2,
+    "traits": [ "FAST_REFLEXES", "LIGHTEATER", "THINSKIN", "FLIMSY" ],
+    "skills": [ { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "runner_bag", "unitard", "lowtops", "socks_ankle", "wristwatch" ],
+        "entries": [ { "group": "charged_cell_phone" } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "panties" ]
+    },
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "runner",
+    "name": "Runner",
+    "description": "Forest Chump was your role model growing up. Time to make him proud.",
+    "points": 2,
+    "traits": [ "GOODCARDIO", "HUNGER", "FLEET", "FLIMSY" ],
+    "skills": [ { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "runner_bag", "b_shorts", "tank_top", "sneakers", "socks_ankle", "wristwatch" ],
+        "entries": [
+          { "group": "charged_cell_phone" },
+          { "group": "charged_mp3" },
+          { "item": "towel" },
+          { "item": "toothbrush_plain" },
+          { "item": "hairbrush" }
```

Comment:
These should be moved out of "entries" and into "items".

Reply:
10-4